cookbooks/aws-parallelcluster-slurm/test/controls/mysql_repo_spec.rb

```diff
@@ -0,0 +1,29 @@
+# Copyright:: 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file.
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+
+control 'mysql_repo_added' do
```

Comment:
should we add `tag:config` as prefix to permit this test to run on CI/CD too?

Reply:
Good catch!
Yes we should. I'll update the PR.